include/quadrature/quadrature_nodal.h

```diff
@@ -0,0 +1,90 @@
+// The libMesh Finite Element Library.
+// Copyright (C) 2002-2019 Benjamin S. Kirk, John W. Peterson, Roy H. Stogner
+
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+
+
+
+#ifndef LIBMESH_QUADRATURE_NODAL_H
+#define LIBMESH_QUADRATURE_NODAL_H
+
+// Local includes
+#include "libmesh/quadrature.h"
+
+namespace libMesh
+{
+
+/**
+ * This class implements nodal quadrature rules for various element
+ * types. For some element types, this corresponds to well-known
+ * quadratures such as the trapezoidal rule and Simpson's rule. For
+ * other element types (e.g. the serendipity QUAD8/HEX20/PRISM15) it
+ * implememens a lower-order-accurate nodal quarature which still
```

Comment:
implements*, quadrature*

Reply:
Wow, I must have had a mini stroke during that sentence.